README.md

```diff
@@ -1,43 +1,33 @@
-[![Build Status](https://travis-ci.org/strinking/docflow.svg?branch=master)](https://travis-ci.org/strinking/docflow)
 # DocFlow
-A Discord Bot written by Members of the [Programming Server](https://discord.gg/010z0Kw1A9ql5c1Qe) to evaluate code and browse documentation.
-
-## To Be Done
-- [ ] Basic Bot framework, using the commands extension of discord.py
-- [ ] Code Evaluation using [Coliru](http://coliru.stacked-crooked.com)
-- [ ] Documentation search for various languages
+A Discord Bot written by Members of the [Programming Server](https://discord.gg/010z0Kw1A9ql5c1Qe) 
+to evaluate code and search documentation for various languages.
 
 ## Setup
-This Project requires a [Python 3.6 Interpreter](https://www.python.org/downloads/) as well as the following packages:
-- discord.py rewrite ([*Documentation*](http://discordpy.readthedocs.io/en/rewrite/))
-
-Usage of a virtual environment is highly recommended.  
-You can setup these using the following commands:
+This Project requires a [Python 3.6 Interpreter](https://www.python.org/downloads/) as well
+as a bunch of dependencies which you can install by using the following command:
 ```bash
-# Create a virtual environment, activate it
-python3 -m venv venv
-source venv/bin/activate
-
-# Install discord.py from GitHub
-python3 -m pip install -U git+https://github.com/Rapptz/discord.py@rewrite
+pip3 install -r requirements.txt
 ```
 
 ## Usage
-You need to set a `DISCORD_TOKEN` environment variable for the Bot to connect to Discord. An easy way to do this by using a Virtual Environment is shown [here](TODO: create wiki page). To start the bot, simply use the following commands:
+*Note: To get started as fast as possible, simply run `make firstrun`*.
 
-```bash
-# If the virtual environment is not activated yet:
-source venv/bin/activate
+You need to set a `DISCORD_TOKEN` environment variable for the Bot to connect to Discord. 
+To start the bot, simply use the following command, from the root directory:
 
-# Run application
-python3 run.py
-
-# When you're done with using it, use `deactivate` to leave the venv
-deactivate
+```bash
+python3 src/bot/run.py
```

Comment:
Yes, you add a `__main__.py` in the top level package. Here's how statbot [does it](https://github.com/strinking/statbot/blob/master/statbot/__main__.py).